api/permissioned.go

```diff
@@ -65,7 +65,7 @@ func permissionedAny(in interface{}, out interface{}) {
 		field := rint.Type().Field(f)
 		requiredPerm := Permission(field.Tag.Get("perm"))
 		if requiredPerm == "" {
-			panic("missing 'perm' tag on " + field.Name)
+			panic("missing 'perm' tag on " + field.Name) // is this okay? can this be used to crash the process?
```

Comment:
This depends only on our code (it verification that we defined perm for all endpoints).